sample/kotlin/src/main/kotlin/com/datadog/android/sample/SampleApplication.kt

```diff
@@ -151,7 +151,7 @@ class SampleApplication : Application() {
                     useCustomEndpoint(BuildConfig.DD_OVERRIDE_SESSION_REPLAY_URL)
                 }
             }
-            .setPrivacy(SessionReplayPrivacy.MASK_USER_INPUT)
+            .setPrivacy(SessionReplayPrivacy.MASK)
```

Comment:
minor: what is the reason for this change?

Reply:
for the test purpose, it shouldn't be in the commit, reverted now